developer/docs/arvo/marks.md

```diff
@@ -7,125 +7,130 @@ title: Marks
 
 # Marks
 
-We've used predefined marks already, but we haven't yet created
-our own marks.  Let's write a sample mark of our own, then chain
-it together with some preexisting ones to have fun with type
-conversions.
+We've used predefined marks already, but we haven't yet created our own marks. 
+Let's write a sample mark of our own, then chain it together with some 
+preexisting ones to have fun with type conversions.
 
-Let's make a small "examples-cord" mark.  "Cord" is a name we use for `@t`
+Let's make a small "cord" mark. "Cord" is a name we use for `@t`
```

Comment:
Maybe link to `++cord` in hoon.hoon, or point to the library section on strings.